### ISSUE TITLE
TraceMap fails for "not-so-faint" fibers. The relative_threshold requirement does not change the result

### ISSUE THREAD

**Issue by gildepaz** [guaix-ucm/megaradrp#273]
TraceMap fails for "not-so-faint" fibers but the relative_threshold requirement does not change the result. Would it be possible to change the trace.py so it reads the relative_threshold requirement? Thx!

**Comment by sergiopasra** [comment]
Yes, `relative_threshold` is not used. I need a test dataset anyway

**Comment by sergiopasra** [comment]
> Yes, `relative_threshold` is not used. 

Well, it's a little more complex than that. It is used if and only if there is no predefined value. But there is a value of `relative_threshold` predefined for all the VPH/INSMODE combination, so the passed value is never used
